ledger/Cargo.toml

```diff
@@ -25,7 +25,8 @@ futures = "0.3.4"
 serde = { version = "1.0", features = ["derive"] }
 thiserror = "1.0.16"
 
-ledger-transport = { git = "https://github.com/Zondax/ledger-rs" }
+# get master once PR has been merged
+ledger-transport = { git = "https://github.com/Zondax/ledger-rs",  branch = "bug/handle-retcode"}
```

Comment:
we can update this now that https://github.com/Zondax/ledger-rs/pull/22 got merged